.github/CHANGELOG.md

```diff
@@ -296,6 +296,11 @@
 
 <h3>Breaking changes</h3>
 
+* Updated how gradient recipes are stored for operations, allowing for
+  gradient recipe definitions involving custom multipliers and with more than
+  two terms.
+  [(#909)](https://github.com/PennyLaneAI/pennylane/pull/909)
```

Comment:
```suggestion
* Updated how parameter-shift gradient recipes are defined for operations, allowing for
  gradient recipes that are specified as an arbitrary number of terms.
  [(#909)](https://github.com/PennyLaneAI/pennylane/pull/909)
  
  Previously, `Operation.grad_recipe` was restricted to two-term parameter-shift formulas.
  With this change, the gradient recipe now contains elements of the form
  $[c_i, a_i, s_i]$, resulting in a gradient recipe of

  $$\frac{\partial}{\partial\phi_k}f(\phi_k) = \sum_{i} c_i * f(a_i * \phi_k+s_i).$$
  
  As this is a breaking change, all custom operations with defined gradient recipes must be
  updated to continue working with PennyLane 0.13. Note though that if `grad_recipe = None`, the
  default gradient recipe remains unchanged, and corresponds to the two terms $[c_0, a_0, s_0]=[1/2, 1, \pi/2]$
  and $[c_1, a_1, s_1]=[-1/2, 1, -\pi/2]$ for every parameter.
```

Reply:
Thought it was best to add some more details! Might have to double check that the math renders correctly.

Reply:
Thanks! Checked and updated.